mysql-test/main/rpl_mysqldump_slave.result

```diff
@@ -65,4 +65,13 @@ SET GLOBAL gtid_slave_pos='0-2-1003';
 -- CHANGE MASTER TO MASTER_LOG_FILE='slave-bin.000001', MASTER_LOG_POS=BINLOG_START;
 CHANGE MASTER TO MASTER_USE_GTID=slave_pos;
 SET GLOBAL gtid_slave_pos='0-2-1003';
+connection master;
+CREATE TABLE t (
+id int
+);
+insert into t values (1);
+insert into t values (2);
+drop table t;
+-- CHANGE MASTER TO MASTER_LOG_FILE='master-bin.000002', MASTER_LOG_POS=BINLOG_START;
```

Comment:
@bnestere  regarding
```
  if (opt_delete_master_logs && !opt_master_data)
    opt_master_data= MYSQL_OPT_MASTER_DATA_COMMENTED_SQL;
```
according to the documentation, the result is good, but according to above code, result should be commented, so that's why I asked where where `opt_master_data` is set to `#define MYSQL_OPT_MASTER_DATA_EFFECTIVE_SQL 1` ?

Reply:
I'm not sure what you mean here. The result is commented (prefixed with `--`). Though I do think the documentation is ambiguous, as it doesn't specify the `opt_master_data` value, so we should extend that to say 
> This option automatically enables --master-data=2.

Reply:
I was thinking that the result shouldn't be commented, by enabling `opt_master_data` set to 1, but that make sense agree, we will update the documentation than. Thanks @bnestere  and @gulshanpr .